perllib/FixMyStreet/Map/BathNES.pm

```diff
@@ -7,12 +7,13 @@ use base 'FixMyStreet::Map::OSM';
 use strict;
 
 sub map_javascript { [
-    '/vendor/OpenLayers/OpenLayers.bathnes.js',
+    '/vendor/OpenLayers/OpenLayers.wfs.js',
```

Comment:
This loses `OpenLayers/Protocol/WFS/v2_0_0.js` - is this needed?

Reply:
oops, yes it is.